Makefile

```diff
@@ -135,6 +135,7 @@ LINK_LIBRARIES = $(foreach i,$(LIBRARIES),-l$(i))
 
 # Default non-gcc opt flags
 DEFAULT_OPT_FLAGS = -Ofast
+SAFETY_OPT_FLAGS = -ftrapping-math -fno-associative-math -fno-unsafe-math-optimizations
```

Comment:
@thecozies did you run these by Wiseguy by chance?

Reply:
yup, removed the last two. they deal with precision more than anything, and with SM64 we don't need that level of precision at the cost of performance. `-ftrapping-math` is really important though, since we have floating point exceptions